cityofaustin/atd-moped
# Synchronize project status between tabs

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/7493
Also I found that deleted statuses were still showing up on the summary page.

## Testing
**URL to test:** 

https://7493-sync-status--atd-moped-main.netlify.app/moped/projects

**Steps to test:**

Add a status on the summary tab
Switch to comments tab and edit status (or delete status)
switch back to summary tab and see that status reflects current state


---
#### Ship list
